mindjunkies/forums/models.py

```
@@ -2,25 +2,29 @@
 from django.db import models
 from django.utils.text import slugify
 
-from mindjunkies.courses.models import Course
-
+from mindjunkies.courses.models import Course, Module
 
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)